src/app/(main)/about/page.tsx

```diff
@@ -0,0 +1,183 @@
+'use client';
+import { Box, Divider, Grid, Link, Paper, Stack, Typography } from '@mui/material';
+import { Variant } from '@mui/material/styles/createTypography';
+
+import { ToolbarPage } from '@respond/components/ToolbarPage';
+import { useAppSelector } from '@respond/lib/client/store';
+
+const APP_BAR_OFFSET = 52;
+
+export default function About() {
+  return (
+    <ToolbarPage maxWidth="lg">
+      <Paper elevation={1} sx={{ p: 3 }}>
+        <Stack direction={{ md: 'row' }} flex="1 1 auto" spacing={1} divider={<Divider orientation="vertical" flexItem />}>
+          <Stack sx={{ mb: 4, minWidth: 175 }}>
+            <AboutMenu />
+          </Stack>
+          <Stack>
+            <AboutOverview />
+            <AboutHomePage />
+            <AboutActivityPage />
+            <AboutResponding />
+            <AboutSartopo />
+          </Stack>
+        </Stack>
+      </Paper>
+    </ToolbarPage>
+  );
+}
+
+function AboutMenu() {
+  const buildId = useAppSelector((s) => s.config.dev.buildId);
+  const buildCommit = buildId.split('-')[0];
+  const buildUrl = buildCommit === 'development' ? '#' : `https://github.com/KingCountySAR/respond-next/commits/${buildCommit}`;
+  return (
+    <>
+      <Typography variant="h6">Respond</Typography>
+      <Typography variant="caption">
+        Build: <Link href={buildUrl}>{buildId}</Link>
+      </Typography>
+      <Typography variant="caption">
+        <Link href="https://forms.gle/VhMeCGUUbUHSw4or9">Submit Feature Request</Link>
```

Comment:
nit: Perhaps this should be "Submit Feedback"? Since we take all feedback there, not just feature requests?